index.js

```diff
@@ -217,6 +217,12 @@ Feed.prototype._open = function (cb) {
     if (state.key) self.key = state.key
     if (state.secretKey) self.secretKey = state.secretKey
 
+    if (self.key && self.secretKey) {
```

Comment:
Adding a comment in front of this if saying that this is to test that the public key corresponds to the secret would be nice to help future collabs grok this :)